lib/angular/directives/fileDetailController.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Controller for file-detail directive
+ *
+ */
+
+function FieldDetailController($scope, mediator){
+  var self = this;
+  self.file = $scope.file;
+  self.options = $scope.displayOptions || {};
+  //publish mediator message to close the file
+  self.closeFile = function(event) {
+    mediator.publish('wfm:file:close:');
```

Comment:
If you are using topics in a module, there should be documentation in the README.md file to describe the topics that the module publishes/subscribes to.

This documentation should also cover the parameters for topics that are published.